.github/issue-templates/linux-nightly-uninstallable.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Linux nightly binaries cannot be installed exclusively from the PyTorch wheel indices
-assignees: pmeier
+assignees: jamesobutler
```

Comment:
@jamesobutler LMK if you don't want to be assigned by the bot in case of CI failures on scheduled runs. No one needs to be assigned here. However, I'd recommend it.

Reply:
I am good with that change.